cmd/konvoy-image/cmd/artifacts.go

```diff
@@ -3,9 +3,8 @@ package cmd
 import (
 	"fmt"
 
-	"github.com/spf13/cobra"
-
 	"github.com/mesosphere/konvoy-image-builder/pkg/app"
+	"github.com/spf13/cobra"
```

Comment:
did a golangci-lint config force us to go from three blocks (stdlib, external, package-local) to two blocks (stdlib and everything-else) ?

Reply:
it was the gci changes.